src/views/SpaceSettings.vue

```diff
@@ -155,14 +155,22 @@ function inputError(field) {
 }
 
 function handleReset() {
-  if (props.from) return (form.value = clone(props.spaceFrom));
-  if (currentSettings.value) return (form.value = currentSettings.value);
-  form.value = {
-    strategies: [],
-    categories: [],
-    plugins: {},
-    filters: {}
-  };
+  try {
+    if (props.from) return (form.value = clone(props.spaceFrom));
+    if (currentSettings.value)
+      return (form.value = clone(currentSettings.value));
+    form.value = {
+      strategies: [],
+      categories: [],
+      plugins: {},
+      filters: {}
+    };
+  } finally {
+    loaded.value = false;
```

Comment:
Looks like an antipattern, if there's a reason to do something like this, its best to write a comment

Reply:
As far as I understood it's a workaround for an issue with the textarea component's state. A comment would be nice though for sure. :D

Reply:
Agree. Added a comment thanks

Reply:
@samuveth Does [this](https://github.com/snapshot-labs/snapshot/pull/1647) maybe help to avoid this workaround?

Reply:
Ok yes let's do that. Looks good btw haven't tested it yet